### ISSUE TITLE
It DELETED all my packages!!!

### ISSUE THREAD

**Issue by xeroxstar** [siddharthkp/auto-install#104]
I installed this module and run in my project as it shown in the tutorial and it deleted all my modules, are you serious? All my modules F... GONE!!!!!!! 

```
× html5sortable removed
× parse-function removed
× eslint removed
× vue2-brace-editor removed
× esprima removed
× decache removed
× auto-install removed
× register-service-worker removed
× splitpanes removed
× vuex removed
- Uninstalling @fortawesome/fontawesome-free
```

and none of them could be installed back by giving me this nonsense:

```
- Installing chokidar
npm ERR! path  ... \node_modules\normalize-path
npm ERR! code ENOENT
npm ERR! errno -4058
npm ERR! syscall rename
npm ERR! enoent ENOENT: no such file or directory, rename
``` 

holly crap, it should say auto-uninstall and not auto-install !!!!

**Comment by OmgImAlexis** [comment]
You had your `package.json` checked into git, no?

**Comment by ceafive** [comment]
Sorry @xeroxstar this happened because the module searched your files and found no explicit 'import' of these modules hence it removed them. I have pushed a commit to make 'uninstall' false on default.

**Comment by siddharthkp** [comment]
@ceafive just a pointer, keep backward compatibility in mind

**Comment by ceafive** [comment]
@siddharthkp yeah got it. This is barely a breaking change I suppose but this solves a critical problem many people might have

**Comment by siddharthkp** [comment]
uninstall is a feature not a bug :)

however, it's supposed to only removed unused packages, not all of them like the issue creator posted
